hostscripts/clouddata/syncSLErepos

```diff
@@ -171,6 +171,7 @@ for version in 9 8 7 6; do
     done
 done
 
+rsync_and_unpack_iso ${ibs_source_nue}/Devel\:/Cloud\:/9\:/Upgrade/images/iso/SUSE-OPENSTACK-CLOUD-CROWBAR-9-x86_64-Media1.iso /srv/nfs/repos/x86_64/SUSE-OpenStack-Cloud-Crowbar-9-devel-upgrade
```

Comment:
It seems this will only be needed in NUE, please only run it there. The UPDATEREPOS idea from @ jsuchome would be even better, then this isn't needed.